devx/action.yml

```diff
@@ -44,7 +49,7 @@ runs:
     - name: Download and import closure
       shell: bash
       run: |
-        ${{ github.action_path }}/support/fetch-docker.sh input-output-hk/devx ${{ inputs.platform }}.${{ inputs.compiler-nix-name }}${{ inputs.target-platform }}${{ inputs.minimal == 'true' && '-minimal' || '' }}${{ inputs.iog == 'true' && '-iog' || '' }}-env | zstd -d | nix-store --import | tee store-paths.txt
+        ${{ github.action_path }}/support/fetch-docker.sh input-output-hk/devx ${{ inputs.platform }}.${{ inputs.compiler-nix-name }}${{ inputs.target-platform }}${{ inputs.minimal == 'true' && '-minimal' || '' }}${{ inputs.iog == 'true' && '-iog' || '' }}${{ inputs.iog-full == 'true' && '-iog-full' || '' }}-env | zstd -d | nix-store --import | tee store-paths.txt
```

Comment:
I'm not sure if this would be a realistic issue or not, but if `iog` and `iog-full` are both true, you'd end up with `-iog-iog-full` which is invalid. You could let one take precedence, eg:
```
${{ (inputs.iog-full == 'true' && '-iog-full') || (inputs.iog == 'true' && '-iog') || '' }}
```